pbench/server/hierarchy.py

```diff
@@ -115,66 +148,16 @@ def dump(self, f):
                 )
                 if check or controller not in self.tarballs:
                     f.write(f"\nController: {controller}\n")
-                    if controller in self.things_to_check["unexpected_dirs"]._dict:
-                        f.write(
-                            "\t* Unexpected state directories found in this controller directory:\n"
-                        )
-                        cnt = self.output_format(f, controller, "unexpected_dirs", cnt)
-                    if controller in self.things_to_check["status_indicators"]._dict:
-                        if (
-                            "subdirs"
-                            in self.things_to_check["status_indicators"]._dict[
-                                controller
-                            ]
-                        ):
-                            f.write(
-                                "\t* No state directories found in this controller directory.\n"
-                            )
-                        cnt += 1
-                    if controller in self.things_to_check["unexpected_symlinks"]._dict:
-                        f.write("\t* Unexpected symlinks in controller directory:\n")
-                        cnt = self.output_format(
-                            f, controller, "unexpected_symlinks", cnt
-                        )
-                    if controller in self.things_to_check["unexpected_objects"]._dict:
-                        f.write("\t* Unexpected files in controller directory:\n")
-                        cnt = self.output_format(
-                            f, controller, "unexpected_objects", cnt
-                        )
+
```

Comment:
Comment: (XXX) in 3.6 dicts are ordered but that's an implementation artifact (for now) so we really should use an ordered Dict.